app/app.js

```diff
@@ -34,7 +34,7 @@ openSansObserver.check().then(() => {
 });
 
 // Import the CSS reset, which HtmlWebpackPlugin transfers to the build folder
-import '../node_modules/sanitize.css/dist/sanitize.min.css';
+import '../node_modules/sanitize.css/sanitize.css';
```

Comment:
What you mean by "I think you should add it manually" ?

Yeah, I am just saying the solution was to add `postcss` here https://github.com/mxstbr/react-boilerplate/blob/v3.0.0/internals/webpack/webpack.base.babel.js#L33 and they are still in global scope.